addons/account_intrastat/data/transport_data.xml

```diff
@@ -1,35 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
   <data noupdate="1">
-    <record id="l10n_be_intrastat.intrastat_trmode_1" model="l10n_be_intrastat.transport_mode">
+    <record id="account_intrastat_transport_1" model="account.intrastat.transport">
       <field name="code">1</field>
```

Comment:
are you sure it's not only valid in Belgium?

Reply:
I found the same codes for Polish:
https://stat.gov.pl/en/intrastat/mode-of-transport-codes/

Germany (page 9):
https://www-idev.destatis.de/idev/doc/intra_en/doc/svz_datei_intra_en.pdf